lib/gooddata/lcm/actions/provision_clients.rb

```diff
@@ -67,7 +67,8 @@ def call(params)
           rescue => e
             params.gdc_logger.error "Problem occurs when provisioning clients. Purge all invalid clients now ..."
             res = PurgeClients.send(:call, params)
-            deleted_client_ids = res.select { |r| r[:status] == 'purged' }.map { |r| r[:client_id] }
+            params.gdc_logger.debug "Purge clients result: #{res}"
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)